src/internet_identity_interface/src/internet_identity/types.rs

```diff
@@ -188,6 +188,13 @@ pub struct AnchorCredentials {
     pub recovery_phrases: Vec<PublicKey>,
 }
 
+/// Init arguments of II which can be supplied on install and upgrade.
+///
+/// Each field is wrapped in `Option<>` to indicate whether the field should
+/// keep the previous value or update to a new value (e.g. `None` keeps the previous value).
+///
+/// Some fields, like `openid_google`, have an additional nested `Option<>`,
+/// this indicates enable/disable status (e.g. `Some(None)` disables a feature).
```

Comment:
Ditto to say that `Some(None) disables a feature whereas None leaves it untouched`